hermeto/core/config.py

```
@@ -119,6 +127,14 @@ def _validate_proxy_url_is_set_when_proxy_credentials_are_set(self) -> Self:
             raise InvalidInput("Proxy URL must be set when proxy credentials are set")
         return self
 
+    @field_serializer("proxy_password")
+    def _serialize_proxy_password(self, value: SecretStr | None, info: Any) -> str | None:
```

Ruff (0.16.0)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `info`

(ANN401)